frontend/src/modules/resourceprogress/components/ResourceProgress.js

```diff
@@ -92,39 +100,39 @@ export class ResourceProgressBase extends React.Component<Props, State> {
                                 Translated
                             </Localized>
                         </span>
-                        <p className="value">{ asLocaleString(approved) }</p>
+                        <p className="value"><a href={ this.handleClickStatusNumbers('translated') } >{ asLocaleString(approved) }</a></p>
```

Comment:
I think these should actually be `react-router` [Links](https://reacttraining.com/react-router/web/api/Link). Look at the examples in the documentation, you can easily manage the same features without using your custom `handleClickStatusNumbers` method.

Reply:
okay on it

Reply:
The `react-router` Links are working good, only that I am facing a blocker with the URL, the base URL is `domain:translate/` so the new URL turns to have `translate` twice in the URL as follows `http://localhost:8000/translate/translate/en-GB/tutorial/all-resources/?status=translated` I am working on finding a fix for this though.

Reply:
Have you tried setting only the query string? Anyway if you want help, the best way would be to show code. :-)

Reply:
Got it. Actually I was not supposed to use the `pathname` property.